test/services/jobs/licence-updates/fetch-licence-updates.service.test.js

```diff
@@ -70,7 +67,9 @@ describe('Fetch Licence Updates service', () => {
       })
 
       it('returns no results', async () => {
-        const results = await FetchLicenceUpdatesService.go()
+        const data = await FetchLicenceUpdatesService.go()
+
+        const results = data.filter((l) => { return l.licenceId === licence.id })
```

Comment:
I am not too happy with this solution. But it does work. 

Maybe the tests descriptions could change to reflect that we are only concerned with the licence we have added for the test ?

Reply:
🎯 I think that is part of the change we need to make in our testing as it will only get harder to avoid clashes as we add more functionality.

When we cleaned before every test it made sense to have tests like, "I added some junk and when I run my service nothing is returned proving it works".

I think we need to move more towards an approach which is more like, "I added some junk and when I run my service _the junk is not returned_ proving it works". Or something! 😁